.github/workflows/inviwo.yml

```diff
@@ -13,39 +13,38 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [windows-latest] #, macos-latest, ubuntu-20.04]
+        os: [windows-latest, macos-latest] #, ubuntu-20.04]
         include:
           - os: 'windows-latest'
             triplet: 'x64-windows'
             mono: ''
-            cmake: '-DIVW_MODULE_HDF5=ON'
-            generator: 'Visual Studio 17 2022'
+            cmake: '--preset msvc-dev-vcpkg -DIVW_MODULE_HDF5=ON -DIVW_DOXYGEN_PROJECT=ON 
+                    -DIVW_DOXYGEN_LATEX_PATH="C:/tools/TinyTeX/bin/win32/pdflatex.exe"'
+            cmake_preset: 'msvc-dev-vcpkg'
```

Comment:
this was never used, since you also put in in the cmake variable...

Reply:
Good catch